src/drivers/github.go

```diff
@@ -24,3 +35,125 @@ func (driver GithubCodeHostingDriver) GetNewPullRequestURL(repository string, br
 func (driver GithubCodeHostingDriver) GetRepositoryURL(repository string) string {
 	return "https://github.com/" + repository
 }
+
+// MergePullRequest merges the pull request through the Github API
+func (driver GithubCodeHostingDriver) MergePullRequest(options MergePullRequestOptions) error {
+	childPullRequestNumbers, err := driver.getPullRequestNumbersAgainst(options.Repository, options.Branch)
+	if err != nil {
+		return err
+	}
+	for _, childPullRequestNumber := range childPullRequestNumbers {
+		err = driver.updatePullRequestBase(options.Repository, childPullRequestNumber, options.ParentBranch)
+		if err != nil {
+			return err
```

Comment:
wrapped error message here: `Cannot update pull request base for #%s via Github API`